### PR TITLE
EN-20312 - Enabled deployment slot tracking on deployment board.

### PR DESCRIPTION
**Pull request by jsclifford**
* Made tracking of slots on deployment board opt in.
* Created new label with `currently-in-env-slotname` label with 🎰`slot_machine` icon  and `blue` color.
* Added inputs to support removing slot label when swapped to production slot.

### CODE REVIEW

**Comment by jsclifford** [comment]
Example of a slot deployed without swap.
image

**Review by JosephDSchwartz — COMMENTED** [review]
This action is being deprecated, so I'm not sure how much it matters, but I don't love including the term `slots` throughout the documentation and code. As far as I know `slots` is a term used by Azure and nothing else.

Is there a more general way of describing what you're trying to accomplish? Like are you trying to indicate code is in a temporary environment (slot) while the real environment is updated? And can it not be accomplished with the existing functionality?

File: src/issues.js [hunk 1]
```diff
@@ -165,12 +173,26 @@ async function createAnIssueForThisDeploymentIfItDoesNotExist(octokit, ghLogin,
...
   core.startGroup(`Appending the deployment details to the issue...`);
 
   let workflowUrl = `[${github.context.workflow} #${github.context.runNumber}](https://github.com/${owner}/${repo}/actions/runs/${github.context.runId})`;
   let nowString = getDateString();
-  let bodyText = `${issueToUpdate.body}\n|${project.columnName}|${workflowUrl}|${deployStatus}|${nowString}|${actor}|`;
+
+  const defaultTableColumns = `|Env|Workflow|Status|Date|Actor|Deploy Label|
+|---|---|---|---|---|---|`;
+  const slotTableColumns = `|Env|Workflow|Status|Date|Actor|Deploy Label|Source Slot|Target Slot|
+|---|---|---|---|---|---|---|---|`;
```
[the remark this answers is not in the document]
Comment (jsclifford) [reply src/issues.js:185]: I just didn't want to have columns on terraform deploys when the feature was disabled.  What do you think.  I can totally default those columns for every deploy and put NA.

[the remark this answers is not in the document]
Comment (JosephDSchwartz) [reply src/issues.js:185]: I think you misunderstood (or I am misunderstanding your reply), these two lines by themselves are fine. The fact that they're duplicated again in the same file (see the link in my original comment) is what rubs me the wrong way.

But honestly I think it's probably fine to merge your changes as-is. This action is deprecated. We can merge this in as a final change.

**Review by JosephDSchwartz — APPROVED** [review]

**Pull request merged**